cmd/vic-init/ops.go

```diff
@@ -83,6 +86,20 @@ func (t *operations) SetHostname(name string, aliases ...string) error {
 	return t.BaseOperations.SetHostname(aliases[0])
 }
 
+func (t *operations) SetupFirewall() error {
+	// get the public interface name
+	l, err := netlink.LinkByName(publicIfaceName)
```

Comment:
This logic seems familiar. Would we want to have a dedicated private function that will perform double lookup?